iina/Base.lproj/Localizable.strings

```diff
@@ -27,7 +27,7 @@
 // Initial Window
 "initial.announcement" = "You are using a %@ version of IINA";
 "initial.beta.desc" = "If you want to rollback to a release version, uncheck \"receive beta updates\" from Settings > General, and install a stable release from our <a href=\"https://iina.io\">official website</a>.";
-"initial.nightly.desc" = "Nightly is not recommended to be used as daily drive. Currently nightly builds do not support auto update. Please manually download newer builds from our <a href=\"https://nightly.iina.io\">nightly website</a>.";
+"initial.nightly.desc" = "Nightly is not recommended to be used as daily driver. Currently nightly builds do not support auto update. Please manually download newer builds from our <a href=\"https://nightly.iina.io\">nightly website</a>.";
```

Comment:
"daily drive" is a back-formation of "daily driver". I too prefer "daily driver". But we are still missing either an "a" or a "your". How about?:

Nightly is not recommended to be used as your daily driver.

Reply:
Oh yes. Amazing I didn't see that! Updated.